deploy/operator/README.md

```diff
@@ -105,7 +105,17 @@ export ASSISTED_AGENT_CLUSTER_INSTALL_NAME=assisted-agent-cluster-install
 export ASSISTED_INFRAENV_NAME=assisted-infra-env
 export ASSISTED_PULLSECRET_NAME=assisted-pull-secret
 export ASSISTED_PRIVATEKEY_NAME=assisted-ssh-private-key
-export SPOKE_CONTROLPLANE_AGENTS=1  # currently only single-node is supported
+export SPOKE_CONTROLPLANE_AGENTS=1  # either 1 (single-node) or 3 is supported
+export SPOKE_WORKER_AGENTS=0 # Number of worker nodes
+
+# JSON encoded dicionary of machine config pool per host.  Example: '{"ostest-extraworker-5":"infra","ostest-extraworker-3":"storage"}'
+export MACHINE_CONFIG_POOLS=machine-config-pools
+
+# JSON encoded dictionary of node-labels per host. Example: '{"ostest-extraworker-3":{"node-role.kubernetes.io/infra":""}, "ostest-extraworker-5":{"node-role.kubernetes.io/infra":""}}'
+export NODE_LABELS=node-labels
+
+# JSON encoded dictionary of manifests to be applied on day1 cluster.  Example: '{"mc.json":"{\"apiVersion\":\"machineconfiguration.openshift.io/v1\",\"kind\":\"MachineConfig\",\"metadata\":{\"labels\":{\"machineconfiguration.openshift.io/role\":\"infra\"},\"name\":\"50-infra\"},\"spec\":{\"config\":{\"ignition\":{\"version\":\"2.2.0\"},\"storage\":{\"files\":[{\"contents\":{\"source\":\"data:,test\"},\"filesystem\":\"root\",\"mode\":420,\"path\":\"/etc/testinfra\"}]}}}}","mcp.json":"{\"apiVersion\":\"machineconfiguration.openshift.io/v1\",\"kind\":\"MachineConfigPool\",\"metadata\":{\"name\":\"infra\"},\"spec\":{\"machineConfigSelector\":{\"matchExpressions\":[{\"key\":\"machineconfiguration.openshift.io/role\",\"operator\":\"In\",\"values\":[\"worker\",\"infra\"]}]},\"maxUnavailable\":null,\"nodeSelector\":{\"matchLabels\":{\"node-role.kubernetes.io/infra\":\"\"}},\"paused\":false}}"}'
```

Comment:
for such a long content, maybe ``MANIFESTS`` should hold file paths that can be read?

Reply:
OK